### PR TITLE
Added new features and optimization [READ EXTENDED DESCRIPTION]

### PR DESCRIPTION
**Pull request by MinecraftPublisher**
Changes made:
1. Removed a few print statements for a bit less cluttering (Makes debugging a bit harder)
2. Added a PRINT command for writing to console.
3. Added an IMPORT command for running external ".dd" files.
4. Added an LED command that toggles the builtin LED.

### CODE REVIEW

**Comment by MinecraftPublisher** [comment]
Also, there seems to be an issue with the stealth mounting on a raspberry pi pico, Is there a fix to this? Whatever I do I can't force my raspberry to not show up as a mass storage device.

**Comment by MinecraftPublisher** [comment]
Another note on the import command, It could be used to loop over multiple actions without having to copy-paste it every time. If I have time, I will make a function to compile ducky script to a python code and compile it to a `.mpy` to make it faster.

**Comment by dbisu** [comment]
Just tested the latest boot.py and seems to be working correctly.

**Comment by MinecraftPublisher** [comment]
Maybe it's a Mac problem? Will try it on Windows just to make sure later.

**Comment by dbisu** [comment]
If you continue to have problems, please open an issue to track it.

**Comment by andreasbrett** [comment]
> Thanks for the fixes on the keycodes. DELETE should be Keycode.DELETE. You are the second person to suggest the IMPORT command. I've been hesitant to include things not in Hak5 DuckyScript for the USB Rubber Ducky. Since DuckyScript hasn't changed for quite a while, maybe it does make sense to go ahead and add these changes. Short answer, fix up the keycode and I'll merge this PR. Thanks.

That's great news! Over on my fork I introduced some further things and also included the mouse jiggler functionality presented by Hak5 a couple of hours ago (https://www.youtube.com/watch?v=aZ8u56I3J3I). If you're interested to merge, let me know.

File: duckyinpython.py [hunk 1]
```diff
@@ -31,6 +32,7 @@
     'INSERT': Keycode.INSERT, 'NUMLOCK': Keycode.KEYPAD_NUMLOCK, 'PAGEUP': Keycode.PAGE_UP,
     'PAGEDOWN': Keycode.PAGE_DOWN, 'PRINTSCREEN': Keycode.PRINT_SCREEN, 'ENTER': Keycode.ENTER,
     'SCROLLLOCK': Keycode.SCROLL_LOCK, 'SPACE': Keycode.SPACE, 'TAB': Keycode.TAB,
+    'BAKCKSPACE': Keycode.BACKSPACE, 'DELETE': Keycode.DELETE,
```
Comment (andreasbrett) [inline duckyinpython.py:35]: typo in `BAKCKSPACE`

Comment (dbisu) [reply duckyinpython.py:35]: Heh, that's what I get for approving PRs at 8am.  I'll get a fix pushed over the weekend.

**Comment by dbisu** [comment]
The mouse jiggler might be a better feature for the FeatherS2 version.  Since that will have Wifi, my thoughts are to include things that could be done remotely.  Keeping a screen awake could be a good feature.

**Comment by MinecraftPublisher** [comment]
I will work on a mouse movement command soon, to allow users to:
```MOUSEPOS 0 0
MOUSEX 10
MOUSE Y -10
LEFTCLICK
MOVEX 5
MOVE Y 10
RIGHTCLICK```

sets the mouse position to 0,0
moves it to 10,0
moves it to 10,-10
left clicks
moves it to 15,-10
moves it to 15,0
right clicks

File: duckyinpython.py (hunk 1 above)
Comment (MinecraftPublisher) [reply duckyinpython.py:35]: Thanks for fixing that, didn't notice it!

Comment (MinecraftPublisher) [reply duckyinpython.py:35]: wait a minute...

**Comment by andreasbrett** [comment]
Sounds interesting. Do you have some scenario in mind where these would come in handy? Just asking because I can't imagine for what attack I would like to integrate "blind" mouse movements/clicking.

**Comment by MinecraftPublisher** [comment]
> Sounds interesting. Do you have some scenario in mind where these would come in handy? Just asking because I can't imagine for what attack I would like to integrate "blind" mouse movements/clicking.

‌The best use I could think of is locking the user out of the computer, If they are dumb enough. More uses would be to bypass security measures that include using a Mouse only, or, the most useless use, keeping your LCD on.

**Comment by andreasbrett** [comment]
Yeah, but how. You don't have any information where to click, nor what resolution the screen has. So how do you manage to click at the right position?

**Comment by MinecraftPublisher** [comment]
> Yeah, but how. You don't have any information where to click, nor what resolution the screen has. So how do you manage to click at the right position?

Maybe I'll add `CLICK_COLOR R G B` to click on the closest designated color, Or just not add it

**Comment by andreasbrett** [comment]
This is a HID device. How would it possibly get display information?

**Comment by MinecraftPublisher** [comment]
Wait right that's not possible, My bad 🥸

**Comment by andreasbrett** [comment]
> The mouse jiggler might be a better feature for the FeatherS2 version. Since that will have Wifi, my thoughts are to include things that could be done remotely. Keeping a screen awake could be a good feature.

Cool, I'll check out what ideas you come up with that new project. With the FeatherS2 you should be able to implement features like on the https://github.com/SpacehuhnTech/WiFiDuck

**Comment by MinecraftPublisher** [comment]
I'll try to implement WiFi actions for my soonly-arriving Nodemcu board, And maybe even bluetooth.

**Comment by andreasbrett** [comment]
> Sounds interesting. Do you have some scenario in mind where these would come in handy? Just asking because I can't imagine for what attack I would like to integrate "blind" mouse movements/clicking.

Found a nice scenario to make use of the mouse movements/clicks. If you need to make it harder for the victim to use their mouse to close down any open e.g. cmd window, move the mouse around every few key presses. Victim won't be able to click in the right places to close your window and needs to resort to keyboard shortcuts which might take longer than the attacking script to finish.

I just implemented the following commands into my fork of pico-ducky:
`MOUSE MOVE $x $y`
`MOUSE WHEEL $amount`
`MOUSE CLICK/PRESS/RELEASE LEFT [RIGHT] [MIDDLE]`
`MOUSE RELEASEALL`

Also nice to open mspaint and draw stuff... ;)

**Comment by MinecraftPublisher** [comment]
Nice!

**Pull request merged**